google/cloud/pubsub/samples/samples.cc

```diff
@@ -936,6 +936,55 @@ void SubscriberConcurrencyControl(std::vector<std::string> const& argv) {
   (argv.at(0), argv.at(1));
 }
 
+void SubscriberFlowControlSettings(std::vector<std::string> const& argv) {
+  namespace examples = ::google::cloud::testing_util;
+  if (argv.size() != 2) {
+    throw examples::Usage{
+        "subscriber-flow-control <project-id> <subscription-id>"};
+  }
+
+  //! [START pubsub_subscriber_flow_settings] [subscriber-flow-control]
+  namespace pubsub = google::cloud::pubsub;
+  using google::cloud::future;
+  using google::cloud::StatusOr;
+  [](std::string project_id, std::string subscription_id) {
+    // Create a subscriber with 16 threads handling I/O work
```

Comment:
Where does the "16 threads" manifest?

Reply:
That was too much cut&paste.  Good thing you asked this, we did not need a custom `Subscriber` for this example, so I was able to simplify it further. PTAL.